lib/MTT/MPI/Install/OMPI.pm

```diff
@@ -78,6 +80,11 @@ sub Install {
     $config->{autogen} = $tmp
         if (defined($tmp));
 
+    $tmp = Value($ini, $section, "ompi_autogen_arguments");
+    $tmp =~ s/\n|\r/ /g;
```

Comment:
I guess we should disregard my comment; I was unfamiliar with the `|` perl regexp operator...